hyrax/README.md

```diff
@@ -4,26 +4,23 @@
 The application contains 
 * Hyrax version 2.3
 * Rails 5.1
-* Ruby version 2.4
+
+Requires:
+* redis
+* Ruby 2.3.1 - 2.5.3 (as tested)
 
 ### Steps to install the application 
-1.  Clone this project 
-2.  To install, cd to the directory and
+1.  Clone this project
+2.  [Set up `rbenv`](https://github.com/rbenv/rbenv#installation) for ruby 2.5.3 with `rbenv install 2.5.3`
+3.  To install, cd to the `hyrax/` directory and
     ```
-    Bundle install
+    gem install bundler
+    bundle install
     bundle exec rake db:migrate
     ```
```

Comment:
Steve, I'd just suggest one small change here. Add something along the lines of you need to have ruby, rails and bundler installed and would include the set up rbenv and gem install bundler under that, as these are one time actions. If you already are a ruby developer, you forget these steps because you have them installed ;).

To install rbenv on Ubuntu, I follow https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-16-04

Thanks for removing the logs, editing .gitignore and adding .ruby-version file. All very useful :)